src/containers/StructurePage/folderComponents/SettingsMenu.tsx

```diff
@@ -7,31 +7,23 @@
  */
 
 import { useTranslation } from "react-i18next";
-import styled from "@emotion/styled";
-import { Root, Trigger, Close, Content, Portal } from "@radix-ui/react-popover";
-import { animations, colors, spacing, stackOrder } from "@ndla/core";
-import { CloseLine } from "@ndla/icons/action";
 import { Settings } from "@ndla/icons/editor";
-import { IconButton } from "@ndla/primitives";
+import { IconButton, PopoverContent, PopoverRoot, PopoverTitle, PopoverTrigger } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
 import { Node } from "@ndla/types-taxonomy";
 import SettingsMenuDropdownType from "./SettingsMenuDropdownType";
-import Overlay from "../../../components/Overlay";
-import RoundIcon from "../../../components/RoundIcon";
 import { getNodeTypeFromNodeId } from "../../../modules/nodes/nodeUtil";
 
-const TitleWrapper = styled.div`
-  display: flex;
-  gap: ${spacing.xsmall};
-`;
-
-const Header = styled.div`
-  display: flex;
-  justify-content: space-between;
-`;
-
-const StyledContent = styled(Content)`
-  z-index: ${stackOrder.dropdown};
-`;
+const StyledPopoverContent = styled(PopoverContent, {
```

Comment:
Brannfakkel, men hva med å bare gjøre dette til en dialog?

Reply:
Jeg prøvde det opprinnelig, men tenkte det var bedre med Popover for å slippe dialog i dialog ..